enhancements/ingress/default-ingress-controller.md

```diff
@@ -0,0 +1,136 @@
+
+This proposal eliminates the contention between the operator and administrators.
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+This proposal eliminates the operator/administrator contention by building on the [User Defined IngressControllers](user-defined-default-ingress-controller.md) capability. Specifically, by:
+
+1. Changing the ingress-operator to stop creating ingresscontrollers altogether
+2. Changing the installer to always render a `default` IngressController
+
+These changes make explicit the notion that the ingress-operator is responsible only for managing IngressControllers created by end users, _not_ for creating IngressControllers.
+
+End users can now assume that the _only_ way IngressControllers are created are via the installer or through their own API calls.
+
+## Motivation
+
+The operator today has a confusing relationship to the `default` IngressController resource. Not only does the operator confound automation in its competition with users for ownership, but the operator is not even guaranteed to have the context necessary to restore any IngressController that was deleted, including the `default` instance. The operator should have a clear responsibility to manage resources brought by the user — the installer being indistinguishable from a user in this context. The installer should have a clear responsibility to always bring the `default` IngressController resource.
+
+### Goals
+
+1. Eliminate contention with users/automation trying to replace the default IngressController post-installation.
+2. Clarify the role of the ingress-operator with respect to IngressController resource creation
+3. Clarify the role of the installer with respect to the default IngressController resource
+4. Provide a well-known file for end-users to modify the default IngressController prior to installation
```

Comment:
A follow-on implication of this proposal is that it becomes more clear how to document for end users how to change the default ingresscontroller during installation: since the installer will always render the resource, docs can refer to the file at its well-known location and say "edit this resource to suit your needs". Today users have to create the file from scratch in the right directory (and that process is undocumented).